content/docs/concepts/csidriver/release/powerflex.md

```diff
@@ -6,18 +6,6 @@ description: Release notes for PowerFlex CSI driver
 
 ## CSI PowerFlex v2.13.0
 
```

Comment:
What is the plan to update the version of respective components (`CSI PowerFlex v2.13.0 -> CSI PowerFlex v2.14.0`) here? It should get updated from a centralized placed, asking as this PR is still referring the older version for all components in their release notes.

Reply:
The action hasn't been run in this PR because for a new GitHub Action, it must be in main. Testing has been done on a forked branch to ensure that the correct files get updated.

That being said, as for the version updates, I will take a further look there. The problem is that we don't know the new version until the repositories are released. We can assume if the version it will but if you have any ideas of knowing prior to them being released/tagged, let me know.

Reply:
@anandrajak1 shared a location where we can retrieve the latest version so I will use that.